tests/pkg/tests/observability_analytics_functional_test.go

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+// Licensed under the Apache License 2.0
+
+package tests
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/types"
+	policyv1 "open-cluster-management.io/governance-policy-propagator/api/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/stolostron/multicluster-observability-operator/tests/pkg/utils"
+)
+
+var k8sClient client.Client
+var once sync.Once
+
+func init() {
+	once.Do(func() {
+		if flag.Lookup("kubeconfig") == nil {
+			flag.String("kubeconfig", "", "Path to the kubeconfig file")
+		}
+	})
+}
```

Comment:
Is this `init` required? I guess it's already initialized at https://github.com/stolostron/multicluster-observability-operator/blob/main/tests/pkg/tests/observability-e2e-test_suite_test.go#L117

Reply:
ACK